src/Text/Pandoc/Writers/JATS.hs

```diff
@@ -69,11 +69,8 @@ docToJATS opts (Pandoc meta blocks) = do
   let (backblocks, bodyblocks) = partition isBackBlock blocks
   let elements = hierarchicalize bodyblocks
   let backElements = hierarchicalize backblocks
-  let colwidth = if writerWrapText opts == WrapAuto
-                    then Just $ writerColumns opts
-                    else Nothing
   let render'  :: Doc -> Text
-      render'  = render colwidth
+      render'  = render Nothing
```

Comment:
Thanks for this PR!  One comment on this line. This disables wrapping entirely.  I don't think we need to do anything that drastic.  Text.Pandoc.Pretty provides a `nowrap` combinator which can locally disable wrapping when that is absolutely needed.  But normal wrapping, which occurs on Space elements, should be innocuous, so you should only need the `nowrap` in a few special cases (e.g. verbatim).

Reply:
Thanks for the review @jgm. I've reinstated the wrapping now.